js/models/contact_model.js

```diff
@@ -361,6 +361,18 @@ angular.module('contactsApp')
 			/* eslint-disable no-console */
 			validate: function(prop, property) {
 				switch(prop) {
+				case 'rev':
+				case 'prodid':
+				case 'version':
+					if (!angular.isUndefined(this.props[prop])) {
+						if(this.props[prop].length > 1) {
```

Comment:
You can put both conditions in a single if statement. That would reduce the code nesting.